.github/workflows/windows-build-and-test.yaml

```diff
@@ -57,6 +58,10 @@ jobs:
           - pg: 14
             pkg_version: ${{ fromJson(needs.config.outputs.pg14_latest) }}.1
             tsl_skips_version: dist_partial_agg-14 dist_grant-14
+          - pg: 15
+            # latest PG15 is not available in chocolatey yet - so use 15.0 for now
+            pkg_version: 15.0.1
```

Comment:
Is this version number correct?? In Postgres we have just two digits (https://www.postgresql.org/support/versioning/)

Reply:
I'm not sure why but the windows package manager uses '.1' as a suffix to standard postgres versions. You can notice the same with pg14's pkg_version as well in line 59.

https://community.chocolatey.org/packages/postgresql